app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Table as NeetoUITable, Typography, Avatar } from "neetoui";
+
+import { CONTACT_TABLE_COLUMN_DATA } from "../constants";
+import { stringToDate } from "../utils";
+
+const Table = ({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+}) => {
+  const modifyContactDateFormat = contacts =>
+    contacts.map(contact => ({
+      ...contact,
+      created_at: stringToDate(contact["created_at"]),
+    }));
+  const renderMenuColumn = () => <MenuHorizontal size={16} />;
```

Comment:
Add a blank between function definitions. Visually that segregation helps to avoid the clutter.

Reply:
Ok Yedhin.